pages/feedback.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Text, Image, Textarea } from "@chakra-ui/react";
+import React, { useState } from "react";
+import Button from "../components/button/Button";
+import StarRating from "../components/starRating/StarRating";
+import feedbackImg from "../public/images/feedbackImg.svg";
+
+const feedback = () => {
+  const [rating, setRating] = useState(0);
+  return (
+    <>
+      <Box pt={"12px"} textAlign={"center"} pb={"15px"}>
+        <Text fontSize={"18px"}>Order Delivered on time!</Text>
```

Comment:
Please add these texts in the en.ts as other strings are added ( for localisation ) and add them in the end so that it can be tracked easily in the file

Reply:
done